catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -1149,4 +1173,16 @@ private static boolean processEntry(String entryName, List<String> requiredAttrs
 
         return dataIsValid.get();
     }
+
+    public static String getMetacardId(Metacard metacard) {
+        String id = metacard.getId();
+
+        Attribute origIdAttr = metacard.getAttribute(MetacardVersion.VERSION_OF_ID);
```

Comment:
Why do we prefer the id of the metacard's history rather than just the metacard.getId()?